gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/ecs/EcsClusterController.groovy

```diff
@@ -34,4 +34,10 @@ class EcsClusterController {
   List all() {
     ecsClusterService.getAllEcsClusters()
   }
+
+  @ApiOperation(value = "Retrieve a list Ecs Cluster Description WRT account and region")
```

Comment:
nit: "Retrieve a list **of** Ecs Cluster Description**s** WRT account and region"